app/initializers/export-application-global.js

```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 import config from '../config/environment';
 
-export function initialize(container, application) {
+export function initialize() {
+  var application = arguments[1] || arguments[0];
```

Comment:
Sounds good. The solution in Ember is to import `isEnabled` and use different signatures based on the flag being enabled. Also, I'm concerned that our tests did not show this (we throw errors if we hit a deprecation).